### ISSUE TITLE
Cant log in In Android version

### ISSUE THREAD

**Issue by cylonr** [hassio-addons/addon-bitwarden#44]
# Problem/Motivation
I have Bitwarden installed on my Home Assistant but suddenly (i don't know if was a update on android version ou HA on the addon-bitwarden ) he stops working on all my android Devices.
The first time e use it, it work perfectly but when i used again, he crash or simply disappear from the screen.
I have the log and i uploaded a movie with the error.
LOGS: https://del.dog/waytonecul.txt
Movie: https://youtu.be/KJDp-aHDxLs

**Comment by madmic1314** [comment]
I had this. Sure there is an issue with the new client version and the version of bw in this. Frenk is waiting for an upstream update before he can resolve.

I pulled this into a separate Docker container and migrated it over. Only thing that was a pain was attachments - not too many.

Either wait for upstream fix or switch seem to be your only options at the moment.

**Comment by CuBiC3D** [comment]
@madmic1314 What's the issue with upstream?

**Comment by Xerionty** [comment]
Can confirm the issue. Started yesterday with my Android device.

**Comment by madmic1314** [comment]
> @madmic1314 What's the issue with upstream?

Not sure - Frenck has mentioned it a few times. He knows what he's talking about so guess we're waiting on that before he can fix this.

Moving to Docker isn't difficult, install Ubuntu Server, Docker, pull bitwardenrs and start. I was a noob and it took me about an hour to get going. Running inside a VM with snapshots makes life much easier. You can export your data from your HA bitwarden and import into the Docker version - all except attachments.

**Comment by lbouriez** [comment]
I just did a fresh install and it works on android last version :/
I have some issues with ios but this is another story

**Comment by damanti-me** [comment]
> > @madmic1314 What's the issue with upstream?
> 
> 
> 
> Not sure - Frenck has mentioned it a few times. He knows what he's talking about so guess we're waiting on that before he can fix this.
> 
> 
> 
> Moving to Docker isn't difficult, install Ubuntu Server, Docker, pull bitwardenrs and start. I was a noob and it took me about an hour to get going. Running inside a VM with snapshots makes life much easier. You can export your data from your HA bitwarden and import into the Docker version - all except attachments.

Attachments worked for me. 
Moved from ha Bitwarden to a docker version on Synology nas.

**Comment by frenck** [comment]
This is fixed in v0.7.0 of the add-on.

<https://github.com/hassio-addons/addon-bitwarden/releases/tag/v0.7.0>
